assimilation_code/modules/utilities/ensemble_manager_mod.html

```diff
@@ -118,26 +118,17 @@ <H2>NAMELIST</H2>
 
 <TR><TD>layout</TD>
     <TD>integer</TD>
-    <TD>Determines the logical process (PE) layout across MPI tasks.
+    <TD>Determines the process (pe) layout on MPI tasks.
      1 is PE = MPI task. 2 is a round-robin layout around the nodes. Layout 2
      results in a more even usage of memory across nodes. This may allow you to run
      with a larger state vector without hitting the memory limit of the node.
      It may give a slight (5%) increase in performance, but this is machine dependent.
-     It has no effect on serial runs.
+     It has no effect on serial runs of filter.
```

Comment:
the ensemble manager is used for more programs than filter so i'd revert this line.

Reply:
tim pointed out all these changes are undoing the update to the Manhattan branch and #46 is doing them to the master branch.  so ignore all these comments.